charts/linkerd-crds/Chart.yaml

```diff
@@ -16,8 +14,7 @@ dependencies:
 - name: partials
   version: 0.1.0
   repository: file://../partials
-# this version will be updated by the CI before publishing the Helm tarball
-version: 0.1.0
+version: 1.0.0-edge
```

Comment:
This gets manually bumped when a patch, minor, or breaking change is made? Since this doesn't need to follow the year.month.edge-number format I think this is fairly straightforward—correct?

Reply:
Correct. This version will change less often than everything else.